src/Juno/JunoClient.ts

```diff
@@ -370,7 +369,6 @@ export class JunoClient {
         name,
         description,
         tags,
-        author,
```

Comment:
>author [](http://example.com/codeflow?start=8&length=6)

make sure you are algining the appropriate deployments on Juno side first.

Reply:
will merge this only after juno changes go live, however juno changes are backward compatible as well.